### ISSUE TITLE
CDefError: cannot parse "CFArrayRef *"  package install

### ISSUE THREAD

**Issue by Zephor5** [wbond/package_control#1390]
keep getting this when running package install

```
Exception in thread Thread-35:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "./package_control/commands/install_package_command.py", line 45, in run
  File "./package_control/package_installer.py", line 58, in make_package_list
  File "./package_control/package_manager.py", line 694, in list_available_packages
  File "./package_control/package_manager.py", line 554, in _list_available
  File "./package_control/package_manager.py", line 453, in list_repositories
  File "./package_control/providers/channel_provider.py", line 201, in get_repositories
  File "./package_control/providers/channel_provider.py", line 87, in fetch
  File "./package_control/download_manager.py", line 340, in fetch
  File "./package_control/downloaders/urllib_downloader.py", line 117, in download
  File "./package_control/downloaders/urllib_downloader.py", line 317, in setup_opener
  File "./package_control/ca_certs.py", line 31, in get_ca_bundle_path
  File "./package_control/ca_certs.py", line 166, in get_system_ca_bundle_path
  File "./package_control/deps/oscrypto/trust_list.py", line 122, in get_path
  File "./package_control/deps/oscrypto/_osx/trust_list.py", line 49, in extract_from_system
  File "./package_control/deps/oscrypto/_ffi.py", line 121, in new
  File "/Library/Python/2.7/site-packages/cffi/api.py", line 257, in new
    cdecl = self._typeof(cdecl)
  File "/Library/Python/2.7/site-packages/cffi/api.py", line 177, in _typeof
    result = self._typeof_locked(cdecl)
  File "/Library/Python/2.7/site-packages/cffi/api.py", line 162, in _typeof_locked
    type = self._parser.parse_type(cdecl)
  File "/Library/Python/2.7/site-packages/cffi/cparser.py", line 476, in parse_type
    return self.parse_type_and_quals(cdecl)[0]
  File "/Library/Python/2.7/site-packages/cffi/cparser.py", line 479, in parse_type_and_quals
    ast, macros = self._parse('void __dummy(\n%s\n);' % cdecl)[:2]
  File "/Library/Python/2.7/site-packages/cffi/cparser.py", line 278, in _parse
    self.convert_pycparser_error(e, csource)
  File "/Library/Python/2.7/site-packages/cffi/cparser.py", line 307, in convert_pycparser_error
    raise CDefError(msg)
CDefError: cannot parse "CFArrayRef *"
<cdef source string>:2:12: before: *
```

**Comment by Zephor5** [comment]
these are debug logs before this error

```
Package Control: Fetching list of available packages and dependencies
  Platform: osx-x64
  Sublime Text Version: 2221
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
  Resolved IPv6: ::ffff:50.116.34.243
Package Control: Urllib Debug Proxy
  http_proxy: http://127.0.0.1:1087
  https_proxy: http://127.0.0.1:1087
  proxy_username: 
  proxy_password: 
Package Control: Generating new CA bundle from system keychain

```

**Comment by wbond** [comment]
Unfortunately Sublime Text 2 does not work on High Sierra or Mojave. You’ll have to use Sublime Text 3.
